external/criu/pingPerf.sh

```diff
@@ -81,7 +81,10 @@ prepare() {
     git clone https://github.com/OpenLiberty/ci.docker.git
     (
         cd ci.docker || exit
-        git checkout instanton
+        openLibertyBranch="instanton"
```

Comment:
This seems like an odd branch...

Reply:
Openliberty `InstantOn` feature (`CRIU`) is still experimental beta version, so it is only available in this branch.

Reply:
a.k.a. "InstantOn (tm)"

;)